src/chat.cpp

```diff
@@ -43,7 +43,7 @@ ChatBuffer::~ChatBuffer()
 {
 }
 
-void ChatBuffer::addLine(std::wstring name, std::wstring text)
+void ChatBuffer::addLine(const std::string &name, const std::string &text)
```

Comment:
Yeah, agreed this has to be changed analogous how  43dab2ffc899133fbd8bb4bf4d209a3bbc70b901 and 3b65a6a36c3e910359c69cd3e3e3fd89e50ba23e did it for `string.h`.

About keeping this wstring: we can't really rely that each `wchar_t` represents an [unicode scalar value](http://www.unicode.org/glossary/#unicode_scalar_value), on some platforms wchars can be part of surrogate pairs too.